mux.go

```diff
@@ -413,6 +406,13 @@ type RouteMatch struct {
 	Handler http.Handler
 	Vars    map[string]string
 
+	// Cleaned version of the path being matched.
```

Comment:
Since these are exported, can you provide clearer guidance for a user trying to understand either:

* What their value is?
* Should I be setting this?
* What the default is?